test/interactor/MemberInteractorTest.java

```diff
@@ -0,0 +1,61 @@
+package interactor;
+
+import boundary.IUser;
+import entity.MemberTask;
+import entity.Profile;
+import entity.Team;
+import gateway.IGateway;
+import model.CreateProfileRequest;
+import model.ProjectTypes.*;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.List;
+
+public class MemberInteractorTest {
+    private class CalledViewProfile extends RuntimeException {}
+
+    private String email;
+    private MemberInteractor memberInteractor;
+
+    private class MockProjectStateManager implements IGateway {
```

Comment:
```suggestion
    private class FakeProjectStateManager implements IGateway {
```

Reply:
Since getProfile takes input and gives output, we use a "Fake". If getProfile did not return a value, we would use a "Mock" to set a value we could use in our test.